gno.land/pkg/sdk/vm/handler.go

```diff
@@ -37,15 +36,7 @@ func (vh vmHandler) Process(ctx sdk.Context, msg std.Msg) sdk.Result {
 
 // Handle MsgAddPackage.
 func (vh vmHandler) handleMsgAddPackage(ctx sdk.Context, msg MsgAddPackage) sdk.Result {
-	amount, err := std.ParseCoins("1000000ugnot") // XXX calculate
-	if err != nil {
-		return abciResult(err)
-	}
-	err = vh.vm.bank.SendCoins(ctx, msg.Creator, auth.FeeCollectorAddress(), amount)
```

Comment:
These should not be removed; rather, MsgAddPkg.Deposit should be parsed above instead of 1000000ugnot. 
The purpose is to deposit the coins to the package itself.
auth.FeeCollectorAddress() is wrong, it should be the package itself.

Reply:
These charges were removed because they were additional to the gas fees specified by users in Gnokey. Previously, because of the absence of a gas meter, we needed this extra fee to prevent misusage. With a gas meter now implemented, the extra charge is no longer needed.

The deposit sent to the realm contract is managed in vm/keeper.go.

https://github.com/gnolang/gno/blob/d54ca62749f79a6992da520e1659ba0dbbdd980b/gno.land/pkg/sdk/vm/keeper.go#L177